docs/guides/backup_security.md

```diff
@@ -1,43 +1,46 @@
-# Backup and security
+# Backup and Security
 
 Security Checklist
 
-- How to backup your account
-- How to restore from a backup
-- How to export a user's Stronghold
-- How to rekey a Stronghold/password rotation
-- Do's and don'ts
+- How to backup your account.
+- How to restore from a backup.
+- How to export a user's Stronghold.
+- How to rekey a Stronghold/password rotation.
+- Do's and don'ts.
 
-## Security checklist
+## Security Checklist
```

Comment:
- If the checklist isn't actually checkable, I'd probably loose the checkboxes.

Reply:
Can be checked in the code but not in the actual page, interesting :D. Removed either way.